compiler/src/main/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/MergeComponentProcessor.kt

```diff
@@ -156,6 +159,7 @@ internal class MergeComponentProcessor(
 
         val componentInterfaces = resolver.getDeclarationsFromPackage(LOOKUP_PACKAGE)
             .filterIsInstance<KSClassDeclaration>()
+            .filter { it.isAnnotationPresent(Origin::class) }
```

Comment:
Creating a class in the file without the origin annotation was causing a failure. Should I not be doing this?